fhir-server/src/main/java/com/ibm/fhir/server/operation/spi/AbstractOperation.java

```diff
@@ -279,4 +284,40 @@ protected FHIROperationException buildExceptionWithIssue(String msg, IssueType i
         OperationOutcome.Issue ooi = FHIRUtil.buildOperationOutcomeIssue(msg, issueType);
         return new FHIROperationException(msg, cause).withIssue(ooi);
     }
-}
+
+    /**
+     * verifies the authorization to an administrative operation.
+     *
+     * @param operation
+     * @param operationContext
+     * @throws FHIROperationException
+     */
+    public void authorize(String operation, FHIROperationContext operationContext) throws FHIROperationException {
+        Object securityContext = operationContext.getProperty(FHIROperationContext.PROPNAME_SECURITY_CONTEXT);
+        Object jwt = operationContext.getProperty(FHIROperationContext.PROPNAME_JWT);
+
+        boolean authorize = false;
+
+        if (jwt != null && jwt instanceof JsonWebToken) {
+            JsonWebToken jwtObj = (JsonWebToken) jwt;
+            Set<String> groups = jwtObj.getGroups();
+            if (groups != null) {
+                for (String group : groups) {
+                    if ("FHIROperationAdmin".equals(group)) {
+                        authorize = true;
+                        return;
+                    }
+                }
+            }
+        }
```

Comment:
I found out that liberty supports mapping JWT subjects to groups, so I don't think we actually need to do something special for JWT authorization.  I think we should just rely on `SecurityContext.isUserInRole` and call it good.

Reply:
Removed